cluster-sync/sync.sh

```diff
@@ -111,6 +112,20 @@ function wait_cdi_pods_updated {
   fi
 }
 
+# Start functional test HTTP server.
+# We skip the functional test additions for external provider for now, as they're specific
+if [ "${KUBEVIRT_PROVIDER}" != "external" ] && [ "${CDI_SYNC}" == "test-infra" ]; then
```

Comment:
Shouldn't we do this when CDI_SYNC is "all" like the default in case CDI_SYNC is somehow not set. Or we should default it to test-infra.